frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -230,6 +232,8 @@ export default {
       try {
         this.user.organizationId = this.userInfo.organizationId
         const response = await ApiService.apiAxios.post(ApiRoutes.USER + '/create', this.user)
```

Comment:
Not a problem for now but we should probably revisit some of our endpoints as they don't follow REST standards. I'm curious how things are in CCOF.